src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -531,7 +531,7 @@ fun processLocalVariables(place: RsElement, processor: (RsPatBinding) -> Unit) {
         processLexicalDeclarations(scope, cameFrom, VALUES) { v ->
             val el = v.element
             if (el is RsPatBinding) processor(el)
-            true
+            false
```

Comment:
I think that this should be false, since with true it only returns the first local variable.